code/game/objects/items/storage/boxes.dm

```diff
@@ -960,6 +960,15 @@
 				return 0
 	return ..()
 
+/obj/item/storage/box/papersack/meat
+	desc = "It's slightly moist and smells like a slaughterhouse."
+
+/obj/item/storage/box/papersack/meat/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/reagent_containers/food/snacks/meat/slab(src)
```

Comment:
why isnt this carp meat?

Reply:
it's meant to be for taming but we don't have it at all so the bag doesn't deserve a place in the kit right now

Reply:
Yep, it has meat with taming in mind, I should as well remove it from the bundle since bee doesn't have it.